### DISCUSSION TITLE
Error in the release notes for 5.6.0

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [mantinedev/mantine#2751] [Q&A]
![image](https://user-images.githubusercontent.com/53124886/196954110-1fdada23-59d3-4b5b-9f99-0c953df4f3df.png)

https://github.com/mantinedev/mantine/releases/tag/5.6.0

**Comment by ArjixWasTaken** [comment]
you import `Radio` but use `Rating`, two different components...

**Comment by rtivital** [comment]
Thanks for reporting

**Comment by ArjixWasTaken** [comment]
np, I love this library!
